### PR TITLE
Minor Spelling Errors and Inconsistencies

### PR DESCRIPTION
**Pull request by DanHarrin**
This PR fixes numerous "oportunity" spelling errors, and inconsistencies between the use of "post process" and "post-process" (of which the latter is more readable).

### CODE REVIEW

File: modules/cms/classes/AutoDatasource.php [hunk 1]
```diff
@@ -100,7 +100,7 @@ public function sourceHasModel(string $source, Model $model)
 
             // Deleted paths are included as being handled by a datasource
             // The functionality built on this will need to make sure they
-            // include deleted records when actually performing sycning actions
+            // include deleted records when actually performing synching actions
```
Comment (LukeTowers) [inline modules/cms/classes/AutoDatasource.php]: https://english.stackexchange.com/a/46473

Comment (DanHarrin) [reply modules/cms/classes/AutoDatasource.php]: so you want that changed back? that source says that both British and Americans prefer `synching`...

Comment (LukeTowers) [reply modules/cms/classes/AutoDatasource.php]: > For similar reasons and consistency, I'd recommend synced over synched and async over asynch.

Yes please

Comment (DanHarrin) [reply modules/cms/classes/AutoDatasource.php]: Done @LukeTowers

**Pull request merged**